openwisp_utils/commitizen/openwisp.py

```
@@ -0,0 +1,181 @@
+        issue_number = match.group(1)
+        return f"{prefix} {title}\n\n" f"{body}\n\n" f"Fixes #{issue_number}"
+
+    def validate_commit_message(
+        self,
+        *,
+        commit_msg: str,
+        pattern: re.Pattern[str],
+        allow_abort: bool,
+        allowed_prefixes: list[str],
+        max_msg_length: int | None,
+        commit_hash: str,
```

Ruff (0.14.14)

[warning] Unused method argument: `commit_hash`

(ARG002)